HongCoin.sol

```diff
@@ -362,6 +363,11 @@ contract TokenCreation is TokenCreationInterface, Token, GovernanceInterface {
                 return;
         }
 
+        if (isFundReleased) {
```

Comment:
This case may not be triggered since there is already a `notReleased` modifier for `createTokenProxy`, which throws when `isFundReleased==true`